pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+## Description
+
+This PR closes #${issue_number}.
```

Comment:
Will it work for GitHub automation? It should add to the board if the statement "Closes #{issue_number} "

Reply:
It will work, the magic work is "closes".